specification/search/data-plane/Azure.Search/stable/2020-06-30/examples/SearchIndexAutocompleteDocumentsGet.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
These aren't ready to go in `stable` yet, since we'll be making post-GA changes. They should also go in the `preview` folder until we consider them complete.

@tg-msft Please correct me if I'm wrong. I'm not sure we've ever used these directories correctly. 😀